reactioncommerce/reaction
# Notification Hover State

Resolves [1980](https://github.com/reactioncommerce/reaction/issues/1980)

**Testing**
- Clone this branch
- Run on Local Machine
- Observe that there is no dark-grey background when hovering on notification icon
